src/olympia/devhub/tests/test_tasks.py

```diff
@@ -567,7 +567,7 @@ def test_run_linter_use_temporary_file_large_output(self):
 
             assert tmpf.call_count == 2
             assert result['success']
-            assert result['warnings'] > 700
+            assert result['warnings'] == 11
```

Comment:
I'll update the comment, you're right. Actually, we're always using a temporary file for the linter output unrelated of it's or the add-ons size for various reasons.